README.md

```diff
@@ -1,105 +1,151 @@
+- [Fedora Messaging 3](https://fedora-messaging.readthedocs.io/en/stable/)
+- [Flask 2](https://flask.palletsprojects.com/)
+- [Flask Caching 2](https://flask-caching.readthedocs.io/en/latest/)
+- [Flask SocketIO 5](https://flask-socketio.readthedocs.io/en/latest/)
+- [Gevent 21](http://www.gevent.org/)
+- [Gunicorn 20](https://gunicorn.org/)
+- [Redis 4](https://github.com/redis/redis-py)
+- [Requests 2](https://requests.readthedocs.io/en/latest/)
 
-You can download the meeting log archive from the link below
+### Development
+- [Black 21](https://black.readthedocs.io/en/stable/)
+- [Flake8 3](https://flake8.pycqa.org/en/latest/)
+- [Isort 5](https://pycqa.github.io/isort/)
+- [Poetry](https://python-poetry.org/)
+- [Pytest 6](https://docs.pytest.org/en/7.1.x/)
+- [Tox 3](https://tox.wiki/en/latest/index.html)
 
-```
-https://mega.nz/file/cJYykbKA#jJozcnIG-WzwlYVQUXF25lqM5A8PNl2knQObQrSpOSk
-```
+## Setup
 
-1. Clone the repository
+### Preliminary setup
+
+1. Fork the repository.
    ```
-   git clone https://github.com/fedora-infra/mote.git
+   https://github.com/fedora-infra/mote
    ```
-2. Navigate to the cloned repository
+2. Download the archive of meeting logs from the link below.
    ```
-   cd mote
+   https://mega.nz/file/cJYykbKA#jJozcnIG-WzwlYVQUXF25lqM5A8PNl2knQObQrSpOSk
    ```
+3. Clone the forked repository to your local storage.
+   ```
+   $ git clone git@github.com:<username>/mote.git
+   ```
+4. Navigate into the cloned repository.
+   ```
+   $ cd mote
+   ```
+5. Depending on the kind of approach required, follow either of the below sections.
 
-### There are two ways to set up this project
-
-#### Containerized setup
-
-3. Install [Podman](https://podman.io/getting-started/installation) (require root privileges) on Fedora
+### Containerized setup
+1. Install [Podman](https://podman.io/getting-started/installation) on Fedora Linux.
    ```
-   sudo dnf install podman
+   $ sudo dnf install podman
    ```
-4. Extract the meetbot archive
+2. Extract the previously downloaded archive of meeting logs in the directory of the cloned repository.
    ```
-   tar xzf meetbot.tar.gz
+   $ tar -xzf meetbot.tar.gz
    ```
-5. Build the container image
+3. Build the container image.
    ```
-   podman build -t mote .
+   $ podman build -t mote .
    ```
-6. Run the server
+4. View the CLI manual for the web service.
    ```
-   podman run -it --rm -p 9696:9696 -v ./meetbot:/srv/web/meetbot:Z mote
+   $ podman run -it --rm mote --help
+   ```
+   ```
+   Usage: mote [OPTIONS]
+   
+   Options:
+     -p, --portdata TEXT  Set the port value [0-65536]
+     -6, --ipprotv6       Start the server on an IPv6 address
+     -4, --ipprotv4       Start the server on an IPv4 address
+     --version            Show the version and exit.
+     --help               Show this message and exit.
+   ```
+4. Start the development server.
+   ```
+   $ podman run -it --rm -p 9696:9696 -v ./meetbot:/srv/web/meetbot:Z mote -p 9696 -4
```

Comment:
That is actually not correct.
The container image is set up to use `gunicorn` as entrypoint, with all the correct parameters.
I don't think there is any need to add the `--help` command in this section since everything is already taken care of.
Just run `podman run -it --rm -p 9696:9696 -v ./meetbot:/srv/web/meetbot:Z -v ./mote:/opt/app/mote:Z mote`.
Note that I've added a volume there `./mote:/opt/app/mote:Z`. That allows you to make modifications locally without the need of rebuilding the container image every time.

Reply:
Perfect. I will make the suggested changes.